app/grpc_server.go

```
@@ -167,6 +167,84 @@ func GetUnaryInterceptor(app *App) grpc.UnaryServerInterceptor {
 	}
 }
 
+// wrappedServerStream lets a StreamServerInterceptor hand a modified context
+// (session, tracing span) down to the actual streaming handler, since
+// grpc.ServerStream.Context() is otherwise read-only.
+type wrappedServerStream struct {
+	grpc.ServerStream
```

golangci-lint (2.12.2)

[error] there must be an empty line separating embedded fields from regular fields

(embeddedstructfieldcheck)